drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -378,6 +378,12 @@ Error AudioDriverWASAPI::audio_device_init(AudioDeviceWASAPI *p_device, bool p_c
 
 		// Due to WASAPI Shared Mode we have no control of the buffer size
 		buffer_frames = max_frames;
+
+		uint32_t output_latency_in_frames = 0;
+		int64_t latency = 0;
+		audio_output.audio_client->GetStreamLatency(&latency);
+		latency = latency * 100; // nanoseconds
```

Comment:
Could you document why you need to multiply by a factor of 100 here? I'm just skimming for errors and noticed that. I think it's normal to deal with factors of 1000 when you're doing math with small amounts of time but multiplying by 100 seems unusual so a more detailed comment would be great.

Reply:
This one is [straight forward from WASAPI](https://docs.microsoft.com/en-us/windows/win32/directshow/reference-time) documentation: unit is 100-nanos. I think the `// nanoseconds` is enough to understand without opening Microsoft documentation?

Reply:
> I think the `// nanoseconds` is enough to understand without opening Microsoft documentation?

Evidently not ;) I didn't understand either without reading the MS docs, so the comment should be made more explicit. E.g.:
```
int64_t latency = 0;
audio_output.audio_client->GetStreamLatency(&latency);
// WASAPI latency unit is 100-ns. Convert to Godot's ms.
real_latency = (float)latency / 1000;
```

And btw my example is not equivalent, the code here seems to convert to seconds. Godot's latency is in ms no?

Reply:
@akien-mga as far as I know, get_output_latency has always returned seconds

Reply:
Alright, edited my snippet to use `10e7` then.